CDCgov/NEDSS-Modernization
# defining profile in useProfileContext to get the latest profile data …

Description 
Fixing a bug in the patient profile where the patients data was not being reflected immediately upon edits or changes done in the demographics card using any table or card where the information is changeable.

## Steps to verify


1. Login
1. Navigate to any patient profile after searching for a patient
2. navigate to demographics and edit to change the "name" of the patient
3. observe the changes reflected immediately in the summary card in the patient profile at the top of the page
